pulsar-broker/src/main/java/com/yahoo/pulsar/broker/service/BrokerService.java

```diff
@@ -972,4 +1018,31 @@ public void onUpdate(String path, Map<String, String> data, Stat stat) {
         return dynamicConfigurationMap;
     }
 
-}
+    private void processPendingLoadTopic() {
+        lock.lock();
+        try {
+            Pair<String, CompletableFuture<Topic>> pendingTopic = pendingTopicLoadingQueue.poll();
+            if (pendingTopic != null) {
+                try {
+                    createPersistentTopic(pendingTopic.getLeft(), true).handle((topic, ex) -> {
+                        if (ex != null) {
+                            pendingTopic.getRight().completeExceptionally(ex);
+                        } else {
+                            pendingTopic.getRight().complete(topic);
+                        }
+                        return null;
+                    });
+                } catch (TooManyRequestsException te) {
```

Comment:
what if this time(as mentioned in above comment) some other new topic3 (from different bundle) comes and acquired a permit and there could be possibility that creation of this topic may fail again with `TooManyRequestsException`. So, we have to again put it back to the queue, as we don't want to fail it and try again.